### ISSUE TITLE
WebApp trains a Hello model

### ISSUE THREAD

**Issue by doyu** [Origami-TinyML/tflm_hello_world#37]

**Comment by ArttuLe** [comment]
Should I still make this for the hello world sine wave, since I started doing the human detection accidentally? :D

**Comment by doyu** [comment]
This really depends on milestone (i.e. demo day). In your case, its 3rd March, where basically you would demonstrate TinyMLaaS end-to-end. If the planned items in Sprint 2 is done, human detection should work end-to-end, which could be a cool demo. If it doesn't meet that, you couldn't demonstrate end-to-end of human detection. This should be discussed with a team. It's a trade-off of project management. If you aim too far, you'll fail. If you aim too near, you miss potential. At least, you need an agreement from a team to see if they could work for that target or not. Please agree on a team how to proceed! Let me know the result, then.

Of course, we (Origami) also have some milestone on 3rd of March, where we'll demonstrate too. we also want a working human detection end-to-end demo there.

**Comment by doyu** [comment]
You may want the progress bar while training:

![Image](https://user-images.githubusercontent.com/3657095/220849204-9903cc65-bd6f-4362-9301-d12419551bdd.png)

**Comment by ArttuLe** [comment]
Yeah, it was also mentioned by the others. I could look into it when polishing the UI of the training section.

**Comment by ArttuLe** [comment]
Also, I have to ensure that the quantized model is working properly. Either the performance is just poor or it isn't working properly, as it was recognizing every picture as "human" regardless of the class, while the non-quantized model was performing correctly. 🤔
